Add browserslist-lint results

a-chabin opened the pull request:
**Changes**
- [x] Added `lint: [ { type, message }, ... ]` field to HTTP API response
_UPD: it seems the server code is not updated in the preview_
- [x] Added output in the client. Create warning, similar errors
- [x] Fixed Node.js examples lint error (https://github.com/browserslist/lint/pull/1)
- [x] Updated README.md

**Questions**
- How to display multiple warnings? (**design**) Now I output with `. <br />`
![image](https://user-images.githubusercontent.com/22644149/186018418-4e3bb4b2-9398-4eca-b388-3544337b1fc6.png)
- Should we do something for warning's a11y?
- Should we automatically update `browserslist-lint` every day (as `browserslist` and `caniuse-lite`)?
- Our first example with `defaults and supports es6-module` contains coverage errors :(

Closes #390

a-chabin commented:
> Package size limit has exceeded by 286 B

I can combine warnings and errors

a-chabin commented:
![image](https://user-images.githubusercontent.com/22644149/186021707-2d2dd8e4-9465-4189-91b1-3ee012305de1.png)

Now we can use this the design.
But he will still have problems with layout shift :(

a-chabin commented:
![image](https://user-images.githubusercontent.com/22644149/186021707-2d2dd8e4-9465-4189-91b1-3ee012305de1.png)

Now we can use this design.
But alerts will have problems with layout shift :(

ai commented:
> How to display multiple warnings? (design) Now I output with . <br />

It is better to add new design.

> Should we do something for warning's a11y?

We will do it as a part of https://github.com/browserslist/browsersl.ist/issues/442

> Should we automatically update browserslist-lint every day (as browserslist and caniuse-lite)?

Good idea. Done https://github.com/browserslist/browsersl.ist/pull/455/commits/a0db0832868989ce98c16f5694bce97723621966

> Our first example with defaults and supports es6-module contains coverage errors :(

Yes, we will think about it in next major update of Browserslist

ai commented:
Note that right now there is an error on result rendering

```
TypeError: o is undefined
```

a-chabin commented:
> Note that right now there is an error on result rendering
Yes. I haven't figured out what the problem is yet.

I wrote about it above.
Locally there is a lint field in HTTP API response, but it is not in the preview.

ai commented:
Why is not updated? We deploy client and server together

a-chabin commented:
> Why is not updated? We deploy client and server together

Fixed! You can watch the preview :)

I forgot about `npm link` when I fixed lint yesterday. Everything is fine with the deployment)

ai commented:
We need to remove "`" from warnings or replace it to `<strong>` ("`" was used to highlight important parts of the warning)

![Снимок экрана от 2022-08-23 10-17-21](https://user-images.githubusercontent.com/19343/186108375-abd9cf04-b2c7-4d3d-a171-2d1af399e8cd.png)

ai commented:
I moved full design to another task https://github.com/browserslist/browsersl.ist/issues/457

a-chabin commented:
Replaced quotes with `<strong>...</strong>`. Looks good!

![image](https://user-images.githubusercontent.com/22644149/186124381-a645043e-ecc3-4125-ab60-e9b877ff7a12.png)
